faucet/miden-faucet.toml

```diff
@@ -0,0 +1,5 @@
+[faucet]
+# port defined as: sum(ord(c)**p for (p, c) in enumerate('miden-store', 1)) % 2**16
```

Comment:
is this updated? just ran it and got a different result:

```python
>>> sum(ord(c)**p for (p, c) in enumerate('miden-store', 1)) % 2**16
28943
```

Also, does it refer to the `endpoint`'s port or the `rpc_url`'s port?

Reply:
Removed the comment.